_rules/text-contrast-afw4f7.md

```diff
@@ -3,7 +3,7 @@ id: afw4f7
 name: Text has minimum contrast
 rule_type: atomic
 description: |
-  This rule checks that the highest possible contrast of every text character with its background meets the minimal contrast requirement.
+  This rule checks that, for text out of widgets, the highest possible contrast of every text character with its background meets the minimal contrast requirement.
```

Comment:
When I first read it, I thought that it was applying to text nodes that are children of elements with the role of the widget. Might be me, but you may consider the other option.

```suggestion
  This rule checks that, for each character in a [text node][] that is not a [child][] of an element with the role of the `widget`, the highest possible contrast with its background meets the minimal contrast requirement.
```

Reply:
I think we usually avoid links in descriptions. What about
```suggestion
  This rule checks that, for text which is not inside a widget, the highest possible contrast of every text character with its background meets the minimal contrast requirement.
```

Reply:
Sounds good!